<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewEntry</class>
 <widget class="QDialog" name="DialogNewEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eintrag erstellen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#DialogNewEntry
{
background-image: url(:/patterns/tex_grey);
background-repeat: repeat-xy;
}

QLabel{
	color: white;
}

QGroupBox {
 border: 1px solid rgb(144,255,23);
 border-radius: 2px; 
 margin-top: 9px;
 margin-bottom: 7px;
 padding: 0px;

background-image: url(:/patterns/woven);
background-repeat: repeat-xy;
} 

QGroupBox::title {
top:-9 ex;
left: 10px;
color: green;
subcontrol-origin: border;

padding: 2px;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;

background-image: 	url(:/patterns/carbon_dark);				
background-repeat: repeat-xy;
}

QPushButton
{
background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;
color: rgb(122, 182, 10);
}

QPushButton:pressed
{

background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(116, 202, 18);
border-style: outset;
color: green;
}

QLineEdit
{
background: rgb(204, 204, 204);
border: 1px solid black;
}



QTabWidget::pane {
	border: 1px solid black;
 }

 QTabWidget::tab-bar {
     left: 2px;
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {

	background-image: 	url(:/patterns/carbon_dark);				
	background-repeat: repeat-xy;

	border-width: 1px;
	border-color:  rgb(0, 102, 0);
	border-style: solid;
	border-bottom: none;
	color: green;
	padding: 4px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     color: rgb(155, 226, 12);
 }

 QTabBar::tab:selected {

	border-width: 1px;
	border-color:  rgb(116, 202, 18);
	border-style: outset;
	border-bottom: none;
	color: rgb(122, 182, 10);
 }

 QTabBar::tab:!selected {
     margin-top: 2px; /* make non-selected tabs look smaller */
 }


QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
 }

 QComboBox:on { /* shift the text when the popup opens */
	 background: yellow;
     padding-top: 3px;
     padding-left: 4px;
 }

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfo">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/patterns/woven);
background-repeat: repeat-xy;</string>
      </property>
      <attribute name="title">
       <string>Informationen</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gpBoxGeneralInfo">
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="title">
          <string>Allgemeine Informationen</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="hozLayoutInfoHeader">
            <property name="topMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="lblEntryName">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lEditEntryName">
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background: rgb(204, 204, 204);
border: 1px solid black;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lblDate">
              <property name="text">
               <string>Datum</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hozLayoutInfoGeneral">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lblAuthor">
                <property name="text">
                 <string>Autor</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lEditAuthor">
                <property name="styleSheet">
                 <string notr="true">background: rgb(204, 204, 204);
border: 1px solid black;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblLanguage">
                <property name="text">
                 <string>Sprache</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="cBoxLanguage">
                <property name="styleSheet">
                 <string notr="true">background: transparent;
border: none;
border-bottom: 1px solid green;
color: white;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <property name="leftMargin">
               <number>37</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item row="2" column="0">
               <widget class="QLabel" name="lblPattern">
                <property name="text">
                 <string>Pattern</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblSystem">
                <property name="text">
                 <string>Betriebsystem</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="cBoxSystem">
                <property name="styleSheet">
                 <string notr="true">background: transparent;
border: none;
border-bottom: 1px solid green;
color: white;</string>
                </property>
                <item>
                 <property name="text">
                  <string>-Keines-</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Windows</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Linux</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Android</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="cBoxPattern">
                <property name="styleSheet">
                 <string notr="true">background: transparent;
border: none;
border-bottom: 1px solid green;
color: white;</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpBoxDescription">
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="title">
          <string>Beschreibung</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPlainTextEdit" name="pTEditDescription">
            <property name="styleSheet">
             <string notr="true">background: transparent;
border: none;
color: white;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCode">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/patterns/woven);
background-repeat: repeat-xy;</string>
      </property>
      <attribute name="title">
       <string>Code</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="title">
          <string>Code Beispiel</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPlainTextEdit" name="pTEditCode">
            <property name="styleSheet">
             <string notr="true">background: transparent;
color: white;
border: none;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hozLayoutFooter">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bttnSpeichern">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
